content/en/docs/intro/install.md

```diff
@@ -21,18 +21,51 @@ Every [release](https://github.com/helm/helm/releases) of Helm provides binary
 releases for a variety of OSes. These binary versions can be manually downloaded
 and installed.
 
-1. Download your [desired version](https://github.com/helm/helm/releases)
-2. Unpack it (`tar -zxvf helm-v3.0.0-linux-amd64.tar.gz`)
-3. Find the `helm` binary in the unpacked directory, and move it to its desired
-   destination (`mv linux-amd64/helm /usr/local/bin/helm`)
+1. Search for and select your desired Helm release from the GitHub [releases](https://github.com/helm/helm/releases)
+2. Download the binary for your platform from the **Installation and Upgrading** section — note that the **Assets** attached to a release contain [archives of the source code](#from-source-linux-macos) and [`*.asc` signature files](#validating-binaries), not the platform binaries
+3. Optionally [validate the integrity and attestation of the binary](#validating-binaries)
+4. Unpack the binary (e.g. `tar -zxvf helm-v3.8.1-darwin-amd64.tar.gz`)
+5. Find the `helm` binary in the unpacked directory, and move it to its desired destination (e.g. `mv darwin-amd64/helm /usr/local/bin/helm`)
 
-From there, you should be able to run the client and [add the stable
-repo](https://helm.sh/docs/intro/quickstart/#initialize-a-helm-chart-repository):
-`helm help`.
+From there, you should be able to run the client (`helm help`) and [add the stable
+repo](https://helm.sh/docs/intro/quickstart/#initialize-a-helm-chart-repository).
 
 **Note:** Helm automated tests are performed for Linux AMD64 only during
 CircleCi builds and releases. Testing of other OSes are the responsibility of
-the community requesting Helm for the OS in question. 
+the community requesting Helm for the OS in question.
+
+#### Validating Binaries
+
+Helm releases include:
+
+- A sha256 checksum (`*.tar.gz.sha256sum`) to validate that the content of the download is what was generated for the release
+- ASCII-armored public keys (`*.asc`) to provide traceability of where the download came from
+
+For more information, please see the [**Release Checklist**](https://helm.sh/docs/community/release_checklist/#8-pgp-sign-the-downloads) documentation.
+
+To validate the integrity and attestation of a downloaded binary:
+
+1. Download the `*.tar.gz.sha256sum` file listed next to the binary you downloaded from the **Installation and Upgrading** section
+2. Download the `*.tar.gz.asc` and `*.tar.gz.sha256sum.asc` signature files that match the platform of your downloaded binary — these can be found in the **Assets** attached to a given [release](https://github.com/helm/helm/releases)
+3. Validate the integrity of the downloaded binary by verifying the sha256 checksum, e.g.
+
+```
+shasum -a 256 helm-v3.8.1-darwin-amd64.tar.gz
+cat helm-v3.8.1-darwin-amd64.tar.gz.sha256sum
+```
+
+4. Validate the attestation of the downloaded binary by cloning the source code repository, importing Helm's `KEYS` file into your keyring, and verifying the signatures, e.g.
```

Comment:
It's also possible to download the keys over HTTPS. What do you think of that?

https://raw.githubusercontent.com/helm/helm/main/KEYS

Reply:
I'll take a look — if I can specify a URL (or the like) as opposed to cloning it down, that would be better — thanks!

Reply:
I've updated this section to:

---

4. Validate the attestation of the downloaded binary by cloning the source code repository, importing Helm's `KEYS` file into your keyring, and verifying the signatures, e.g.


```
> $ curl --show-error --silent https://raw.githubusercontent.com/helm/helm/main/KEYS | gpg --import -
> $ gpg --verify helm-v3.8.1-darwin-amd64.tar.gz.asc helm-v3.8.1-darwin-amd64.tar.gz
> $ gpg --verify helm-v3.8.1-darwin-amd64.tar.gz.sha256sum.asc helm-v3.8.1-darwin-amd64.tar.gz.sha256sum
```

---

https://github.com/Timothy-Dement/helm-www/blob/d320d2606fbfa5846005b29121e9c3423f91d60e/content/en/docs/intro/install.md?plain=1#L59-L65